<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerMainWidget</class>
 <widget class="QWidget" name="PlayerMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="XPlayerScreenWidget" name="ScreenWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>101</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="openFileButton">
      <property name="minimumSize">
       <size>
        <width>27</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>27</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/images/openfile.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/images/play.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>118</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>00:00:00/00:00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="playProgressSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>560</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::add-page:Horizontal{
background-color: rgb(87, 97, 106);
height:3px;}
QSlider::sub-page:Horizontal {
background-color:rgb(40, 115, 216);
height:3px;}
QSlider::groove:Horizontal{
background:transparent;
height:3px;}
QSlider::handle:Horizontal{
height: 12px;
width:12px;  
border-image: url(:/res/images/slider_handle.png);
margin: -4 -2px;  }</string>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>3</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XPlayerScreenWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">xplayerscreenwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xplayer.qrc"/>
 </resources>
 <connections/>
</ui>
